System.Compression.Primitives/System/Interop/Interop.Brotli.x64.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
```

Comment:
All these files need the license notice. See other files in corfxlab as an example.